### PR TITLE
Add invoices plugin migration guide to the app

### PR DESCRIPTION
**Pull request by lkostrowski**

### CODE REVIEW

**Comment by 2can** [comment]
@lkostrowski please fix the `Shop settings` link

File: docs/developer/app-store/apps/invoices.mdx [hunk 1]
```diff
@@ -83,3 +83,22 @@ Visit the [development guide](../../extending/apps/developing-apps/app-examples)
 Apart from common env variables, the Invoice app accepts the following:
 
 - `TEMP_PDF_STORAGE_DIR=` - a temp directory where the PDF files will be stored. In Vercel, set `/tmp`. Add it to `.gitignore`
+
+## Migration from the _Invoicing_ plugin
+
+The invoices app covers the functionality of the plugin. In the future, plugins will be deprecated and replaced by apps. To migrate from plugin to app, follow the steps below:
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:89]: **suggestion (non-blocking)**:
link to the plugin docs

Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:89]: **polish:**

```suggestion
The Invoices App covers the functionality of the plugin. In the future, plugins will be deprecated and replaced by apps. To migrate from plugin to app, follow the steps below:
```

File: docs/developer/app-store/apps/invoices.mdx [hunk 2]
```diff
@@ -83,3 +83,22 @@ Visit the [development guide](../../extending/apps/developing-apps/app-examples)
...
+
+1. Install the Invoices App from the App Store (Dashboard -> Apps).
+2. Open the installed App.
+3. Verify the default shop billing address. It will be fetched from [Shop Settings](<(../../../api-storefront/miscellaneous/objects/shop)>).
+4. Verify per-channel shop address settings. It will use the default one but can be overridden.
+5. Open a test order in the Dashboard.
+6. Click the "Generate" button in the "Invoices" section.
+7. Wait for two PDF files should be generated - one from the plugin and the second from the app. You may need to refresh the page.
```
Comment (peelar) [inline docs/developer/app-store/apps/invoices.mdx:97]: **polish:**

```suggestion
7. Wait for two PDF files to generate: one from the plugin and the second from the app. You may need to refresh the page.
```

**Review by peelar — APPROVED** [review]

File: docs/developer/app-store/apps/invoices.mdx (hunk 1 above)
Comment (lkostrowski) [reply docs/developer/app-store/apps/invoices.mdx:89]: There is no docs for this plugin

**Comment by lkostrowski** [comment]
@2can fixed

**Pull request merged**